docs/docs/studio.md

```diff
@@ -89,37 +201,82 @@ token.
 
 Requires a `SUPABASE_JWT_SECRET` environment variable for JWT signing.
 
-`redwood.toml` example:
-
 ```toml
+// redwood.toml
+
 [studio.graphiql.authImpersonation]
   authProvider = "supabase"
   email = "user@example.com"
   userId = "1"
 ```
 
-### Settings
-
-All settings for Studio are located in `redwood.toml`, which you can find at
-the root of your Redwood project.
-
-* `[studio.graphiql.authImpersonation].*` – Used to gain access to GraphQL
-  endpoints that require authentication. See section above on auth
-  impersonation for more details.
-* `[studio].basePort` – Studio's web front-end will run on this port (default:
-  4318). It is also used to calculate the port for the mailer integration and
-  other things. Please choose a port that is not already in use, and that has a
-  few more free ports available next to it.
-
 ### Database File 
 Studio stores the ingested telemetry to `studio/prisma.db` within the
 `.redwood` folder. You should not need to touch this file other than if you
 wish to delete it to erase any existing telemetry data.
 
-## Availability
-Along the release of Redwood v7, Studio has been rewritten and is available as
-a stable version. Just run `yarn rw studio` to start it! Prior to RW v7 Studio
-was available as an experimental feature. If you're still not using the stable
-version of RW v7, see this forum topic for information on the old experimental
-version of Studio
-https://community.redwoodjs.com/t/redwood-studio-experimental/4771
+## Troubleshooting
```

Comment:
I think we can remove this section because this applies to every single page in the docs. I think we've already done a good job of informing people where to go if they have an issue.